dev_guide/event_notifications.adoc

```diff
@@ -0,0 +1,98 @@
+[[dev-guide-event-notifications]]
+= Event Notifications
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+For each of your projects, you can choose to receive email notifications about
+various xref:../dev_guide/events.adoc#dev-guide-events[events], including:
+
+* _Dead deployment_: A failed deployment in which no other deployment in the
+rollout history is able to remain active. The related deployment configuration
+has no deployments with active replicas.
+
+* _Failed deployment_: A new deployment fails and another deployment in the
+rollout history is now active. The related deployment configuration has one
+active deployment, but it is not the latest deployment.
+
+* _Stuck deployment_:  A deployment in which the active replicas are less than
+ the replicas defined by the deployment configuration for a certain amount of
+ time.
```

Comment:
Hrmm well, it is configurable. I don't think we've decided on a specific amount of time for it yet. Maybe something like "for an unreasonable amount of time" will do for now?